packages/shared/src/components/cards/list/CardCover.tsx

```diff
@@ -23,28 +22,18 @@ export function CardCoverList({
   post,
   className,
 }: CardCoverProps): ReactElement {
-  const { shouldShowOverlay, onInteract } = usePostShareLoop(post);
-  const coverShare = (
-    <CardCoverShare
-      post={post}
-      onShare={() => {
-        onInteract();
-        onShare(post);
-      }}
-      onCopy={onInteract}
-    />
-  );
+  const { overlay } = useCardCover({ post, onShare });
```

Comment:
More of a generic question, what are the diffs between this one on list/card?
Wondering if maybe we can make all the shared stuff/actions uniform better.

Reply:
After checking, the only difference seems to be with the `wrapper` for the list mode. Not sure if there was any reason behind adding almost the same thing. Probably to avoid any unnecessary change on card that affects list.

Reply:
So maybe we can abstract the inner to a common file? 🙏

Reply:
Yeah, makes sense. We can abstract it even further. Let me check quickly.

Reply:
Refactored and using the same component now 🚀